lib/ansible/modules/system/java_certs.py

```diff
@@ -0,0 +1,250 @@
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+DOCUMENTATION = '''
+---
+module: java_cert
+version_added: 2.3
+short_description: Uses keytool to import/remove key from java keystore(cacerts)
+description:
+  - This is a wrapper module around keytool. Which can be used to import/remove certificates from a given java keystore.
+options:
+  cert_url:
+    description:
+      - Basic URL to fetch SSL certificate from
+    required: true
+  cert_port:
+    description:
+      - Port to connect to URL. This will be used to create server URL:PORT
+    required: false
+    default: 443
+  cert_path:
+    description:
+      - Local path to load certificate from
+    required: false
+    default: null
+  cert_alias:
+    description:
+      - Imported certificate alias
+    required: false
+    default: null
+  keystore_path:
+    description:
+      - Path to keystore.
+    required: false
+    default: null
+  keystore_pass:
+    description:
+      - Keystore password
+    required: false
+    default: null
+  keystore_create:
+    description:
+      - Create keystore if it doesn't exist
+    required: false
+    default: null
+  state:
+    description:
+      - Defines action which can be either certificate import or removal.
+    choices: [ 'present', 'absent' ]
+    default: present
+    required: false
+
+author: Adam Hamsik @haad
+'''
+
+EXAMPLES = '''
+# Import SSL certificate from google.com to a given cacerts keystore
+java_certs:
+  cert_url: google.com
+  cert_port: 443
+  keystore_path: /usr/lib/jvm/jre7/lib/security/cacerts
+  keystore_pass: changeit
+  state: present
+
+# Remove certificate with given alias from a keystore
+java_certs:
+  cert_url: google.com
+  keystore_path: /usr/lib/jvm/jre7/lib/security/cacerts
+  keystore_pass: changeit
+  state: absent
+
+# Import SSL certificate from google.com to a keystore, create it if it doesn't exist
+java_certs:
+  cert_url: google.com
+  keystore_path: /tmp/cacerts
+  keystore_pass: changeit
+  keystore_create: yes
+  state: present
+'''
+
+RETURN = '''
+msg:
+  description: Output from stdou of keytool command after execution of given command.
+  returned: success
+  type: string
+  sample: "Module require existing keystore at keystore_path '/tmp/test/cacerts'"
+
+rc:
+  description: Keytool command execution return value
+  returned: success
+  type: int
+  sample: "0"
+
+cmd:
+  description: Executed command to get action done
+  returned: success
+  type: string
+  sample: "keytool -importcert -noprompt -keystore"
+'''
+
+import os
+
+def check_cert_present(module, keystore_path, keystore_pass, alias):
+    ''' Check if certificate with alias is present in keystore located at keystore_path '''
+    test_cmd = ("keytool -noprompt -list -keystore '{}' -storepass '{}' "
+                "-alias '{}'").format(keystore_path, keystore_pass, alias)
```

Comment:
To provide better supports for older version of Python (like 2.5), it's better not to use `format()` but to use `%`-style formatting.
Also in L131, 133, 150, 164, 188 & 224.